http-client/src/main/java/io/micronaut/http/client/netty/PoolResizer.java

```diff
@@ -221,6 +222,10 @@ final void onConnectionInactive2(ResizerConnection connection) {
     }
 
     final void addPendingRequest(Sinks.One<ConnectionManager.PoolHandle> sink) {
+        if (pendingRequests.size() >= connectionPoolConfiguration.getMaxPendingAcquires()) {
+            sink.tryEmitError(new HttpClientException("Cannot acquire connection, exceeded max pending acquires configuration"));
+            return;
+        }
```

Comment:
Do we have a config test for maxPendingAcquires and `max-concurrent-http1-connections`?

Reply:
ive added tests for both. i dont like the pure config tests like in DefaultHttpClientConfigurationSpec